src/dtgtk/thumbtable.c

```diff
@@ -2009,6 +2010,30 @@ static gboolean _accel_rate(GtkAccelGroup *accel_group, GObject *acceleratable,
 {
   GList *imgs = g_list_copy((GList *)dt_view_get_images_to_act_on(FALSE, TRUE));
   dt_ratings_apply_on_list(imgs, GPOINTER_TO_INT(data), TRUE);
+
+  // if we are in darkroom we show a message as there might be no other indication
+  const dt_view_t *v = dt_view_manager_get_current_view(darktable.view_manager);
+  if(v->view(v) == DT_VIEW_DARKROOM && g_list_length(imgs) == 1 && darktable.develop->preview_pipe)
+  {
+    // we verify that the image is the active one
+    const int id = GPOINTER_TO_INT(g_list_nth_data(imgs, 0));
+    if(id == darktable.develop->preview_pipe->output_imgid)
+    {
+      const dt_image_t *img = dt_image_cache_get(darktable.image_cache, id, 'r');
+      if(img)
+      {
+        const int r = img->flags & DT_IMAGE_REJECTED ? DT_VIEW_REJECT : (img->flags & DT_VIEW_RATINGS_MASK);
+        dt_image_cache_read_release(darktable.image_cache, img);
+
+        // translate in human readable value
+        if(r == DT_VIEW_REJECT)
+          dt_toast_log(_("image rejected"));
+        else
+          dt_toast_log(ngettext("image rated to %d star", "image rated to %d stars", r), r);
```

Comment:
BTW, we would also display a number of ★ corresponding to the rating. Should be easy with a simple table of strings.